### PR TITLE
Add libgpg-error

### PR DESCRIPTION
**Pull request by martindurant**

### CODE REVIEW

**Comment by mrocklin** [comment]
This has been stalled for a while.  Is there anything we can do to push it forward?

File: recipes/libgpg-error/build.sh [hunk 1]
```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+
+./configure --prefix=$PREFIX
+
+make
+make install
```
Comment (ocefpaf) [inline recipes/libgpg-error/build.sh]: Is there a `make check` we can run here?

Comment (martindurant) [inline recipes/libgpg-error/build.sh]: Not without increasing the build requirements and complexity (gtest, gmock), which I couldn't actually get working.

Comment (ocefpaf) [inline recipes/libgpg-error/build.sh]: Can you open an issue about that in the feedstock as soon as it gets created?

Comment (martindurant) [inline recipes/libgpg-error/build.sh]: OK

**Comment by mrocklin** [comment]
Thank you for the push @ocefpaf !

**Pull request merged**